### PR TITLE
Feature/1152 jobs facetview

### PR DESCRIPTION
**Pull request by richard-jones**
Adds the UI for browsing background jobs

### CODE REVIEW

**Comment by Steven-Eardley** [comment]
At the moment, the IDs list goes a long way off to the side, and the selection_query hasn't been parsed:


screen shot 2016-12-28 at 11 23 29

File: portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ jQuery(document).ready(function($) {
             paramsBlock += "<strong>Parameters:</strong><br>";
             for (var key in resultobj.params) {
                 var val = resultobj.params[key];
+                if (val instanceof Array) {
```
Comment (richard-jones) [inline portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js]: I've never used instanceof before, so I'm not sure of the compatibility of it.  There is a $.isArray function in jquery, which makes me suspicious of instanceof.  Anyone know?  A quick google hasn't turned up anything obviously pro or against this approach.

File: portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js [hunk 2]
```diff
@@ -53,7 +56,7 @@ jQuery(document).ready(function($) {
         if (resultobj.reference) {
             paramsBlock += "<strong>Reference:</strong><br>";
             for (var key in resultobj.reference) {
-                var val = resultobj.reference[key];
+                var val = JSON.stringify(resultobj.reference[key]);
```
Comment (richard-jones) [inline portality/static/doaj/js/available_facetviews/admin.background_jobs.facetview.js]: This puts quote marks around the value when it's a string, which might be confusing, so I may add a catch for arrays and objects and only use JSON.stringify in those cases.

**Comment by richard-jones** [comment]
I've merged this for convenience in other bits of the project.  We can add any further fixes to it through other branches.

**Comment by emanuil-tolev** [comment]
looks good anyway

**Pull request merged**